src/goCheck.ts

```diff
@@ -106,6 +106,43 @@ function runTool(args: string[], cwd: string, severity: string, useStdErr: boole
 	});
 }
 
+/**
+ * Check if the given path is a directory.
+ * @param filePath Path to the potential directory
+ */
+function isDirectory(filePath: string): boolean {
+	try {
+		return fs.statSync(filePath).isDirectory();
+	} catch (e) {
+		return false;
+	}
+}
+
+/**
+ * List directories (including the given dirPath) which contain files with the '_test.go' suffix.
+ * This ignores hidden directories and their sub-directories.
+ * @param dirPath Directory in which to start searching
+ */
+function findTestDirectories(dirPath: string): string[] {
```

Comment:
> go test -run ! ./...

Lets give this a try. I was initially not in favor of this as the output is different (lots of "no test files" followed by a "build failed" on the pkg which has error) and we will have to write a different regex to do the problem matching. 

But taking a closer look, there is also a line in there which matches our expectation and our current regex should work.

<img width="791" alt="screen shot 2017-06-08 at 12 12 22 pm" src="https://user-images.githubusercontent.com/16890566/26946345-c6383e54-4c43-11e7-9df3-4c4984e5b592.png">